ansys/dpf/core/operators/averaging/elemental_difference.py

```diff
@@ -7,43 +7,58 @@
 from ansys.dpf.core.outputs import Output, _Outputs, _modify_output_spec_with_one_type
 from ansys.dpf.core.operators.specification import PinSpecification, Specification
 
-"""Operators from Ans.Dpf.FEMutils plugin, from "averaging" category
-"""
+"""Operators from Ans.Dpf.FEMutils plugin, from the "averaging" category."""
 
 class elemental_difference(Operator):
-    """Transform ElementalNodal or Nodal field into Elemental field. Each elemental value is the maximum difference between the computed result for all nodes in this element. Result is computed on a given element scoping.
-
-      available inputs:
-        - field (Field, FieldsContainer)
-        - mesh_scoping (Scoping) (optional)
-        - mesh (MeshedRegion) (optional)
-        - through_layers (bool) (optional)
-
-      available outputs:
-        - fields_container (FieldsContainer)
-
-      Examples
-      --------
-      >>> from ansys.dpf import core as dpf
-
-      >>> # Instantiate operator
-      >>> op = dpf.operators.averaging.elemental_difference()
-
-      >>> # Make input connections
-      >>> my_field = dpf.Field()
-      >>> op.inputs.field.connect(my_field)
-      >>> my_mesh_scoping = dpf.Scoping()
-      >>> op.inputs.mesh_scoping.connect(my_mesh_scoping)
-      >>> my_mesh = dpf.MeshedRegion()
-      >>> op.inputs.mesh.connect(my_mesh)
-      >>> my_through_layers = bool()
-      >>> op.inputs.through_layers.connect(my_through_layers)
-
-      >>> # Instantiate operator and connect inputs in one line
-      >>> op = dpf.operators.averaging.elemental_difference(field=my_field,mesh_scoping=my_mesh_scoping,mesh=my_mesh,through_layers=my_through_layers)
-
-      >>> # Get output data
-      >>> result_fields_container = op.outputs.fields_container()"""
+    """Transform an ``ElementalNodal`` or ``Nodal`` field into an ``Elemental`` field. 
+    
+    Each elemental value is the maximum difference between the computed result for all 
+    nodes in this element. The result is computed on a given element scoping.
+
+    Parameters
+    ----------
+    field : str, optional
+        Name of the field or fields container. The default is ``None``.
+    mesh_scoping : optional
+        Name of the mesh scoping. The default is ``None``.
+    mesh : optional
+        Name of the meshed region. The default is ``None``.
+    through_layers : bool, optional
+    
+    config : optional
+        The default is ``None``.
+    server : optional
+        The default is ``None``.
+        
+    Returns
+    -------
+    :class:`ansys.dpf.core.fields_container`
```

Comment:
I was told that classes do not return anything--yet you had an "available outputs" section. Consequently, I formatted as a Returns section--using the class directive that I was taught and then a general description for the return type.

Reply:
Returns for classes should be removed.  If these are attributes, we should be using an [Attributes](https://numpydoc.readthedocs.io/en/latest/format.html#class-docstring) Section.